src/test/java/org/jsoup/parser/TemplateTest.java

```diff
@@ -0,0 +1,60 @@
+package org.jsoup.parser;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TemplateTest {
+    @Test
+    public void testParseTemplateWidthTd() {
+        String html = "<template id=\"lorem-ipsum\">\n" +
+                "    <td>Lorem</td>\n" +
+                "    <td>Ipsum</td>\n" +
+                "</template>";
+
+
+        Document doc = Jsoup.parse(html);
+        doc.outputSettings().prettyPrint(false);
+        doc.outputSettings().syntax(Document.OutputSettings.Syntax.xml);
+
+        assertEquals("<html><head></head><body><template id=\"lorem-ipsum\">\n" +
```

Comment:
Maybe instead of checking HTML literally you should verify that `doc.body()` is `template` tag and has two `td` tags as children?